### PR TITLE
Active record

### PR DESCRIPTION
**Pull request by Sboursen**
# Description

Milestone #4

- [x] Create Models
- [x] Use Models to insert data
- [x] Create custom methods

Cheers,

### CODE REVIEW

File: app/models/like.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User',
+                      foreign_key: 'author_id'
+  belongs_to :post, counter_cache: :likes_count
+end
```
Comment (ShoiraTa) [inline app/models/like.rb]: - [x] Like model should have a method that updates the likes counter for a post: 

```ruby
def update_likes_counter
  post.increment!(:likes_counter)
end
```

File: app/models/comment.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User',
+                      foreign_key: 'author_id'
+  belongs_to :post, counter_cache: :comments_count
+end
```
Comment (ShoiraTa) [inline app/models/comment.rb]: - [x] Comment model should have a method that updates the comments counter for a post.
```ruby
def update_comments_counter
  post.increment!(:comments_counter)
end
```

**Review by ShoiraTa — APPROVED** [review]
Hi @Sboursen ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- Great commit history! ✔️ 
- You passed all the Linters! ✔️ 
- Professional pull request! ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Sboursen** [comment]
@ShoiraTa 

Nice catch!
Thanks for the feedback

**Comment by ShoiraTa** [comment]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @Sboursen , 

**Great work on making the changes 👏🏻** 

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**